.github/workflows/build.yml

```diff
@@ -865,6 +865,27 @@ jobs:
           makeLatest: true
           tag: ${{ steps.version.outputs.version }}
           token: ${{ secrets.NETDATABOT_GITHUB_TOKEN }}
+      - name: Checkout netdata main Repo # Checkout back to netdata/netdata repo to the update latest packaged versions
+        id: checkout-netdata
+        if: steps.create-release.output.run == 'true'
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.NETDATABOT_GITHUB_TOKEN }}
+      - name: Update latest packaged version
+        id: create-latest-versions-metadata
+        if: steps.checkout-netdata.output.run == 'true'
+        run: |
+          sh .github/scripts/update-latests-per-major-version.sh $(cat packaging/version)
+      - name: Commit Changes
+        id: commit-latest-releases
+        if: steps.create-latest-versions-metadata.outputs.run == 'true'
+        env:
+          GITHUB_TOKEN: ${{ secrets.NETDATABOT_GITHUB_TOKEN }}
+        run: |
+          git config user.name "netdatabot"
+          git config user.email "bot@netdata.cloud"
+          git add packaging/agent-metadata/nightly
+          git commit -m "[ci skip] Update the latest major version published"
```

Comment:
This also produces no changes on master, since nothing gets pushed.

Reply:
Addressed in https://github.com/netdata/netdata/pull/16366/commits/472a496c6dbeab6c2350df945e67afbb09a4cf6f